mowgli_etl/pipeline/wdc/wdc_heuristic_product_type_classifier.py

```diff
@@ -5,13 +5,24 @@
 
 
 class WdcHeuristicProductTypeClassifier(WdcProductTypeClassifier):
-    def classify(self, *, title: str) -> WdcProductType:
+    NLP = load('en_core_web_sm')
```

Comment:
I'd probably stash this somewhere since you're using it in multiple places. Just make a module that exports it as a variable?